python/cugraph/traversal/sssp.pxd

```diff
@@ -18,10 +18,12 @@
 
 from cugraph.structure.graph_primtypes cimport *
 
-cdef extern from "algorithms.hpp" namespace "cugraph":
+cdef extern from "utilities/cython.hpp" namespace "cugraph::cython":
 
-    cdef void sssp[VT, ET, WT](
-        const GraphCSRView[VT, ET, WT] &graph,
+    cdef void call_sssp[VT, WT](
```

Comment:
We'd better replace VT, ET, WT with vertex_t, edge_t, weight_t. Please search for this.

Reply:
updated